public/app/core/components/Select/FolderPicker.tsx

```diff
@@ -54,7 +54,10 @@ export interface Props {
   skipInitialLoad?: boolean;
   /** The id of the search input. Use this to set a matching label with htmlFor */
   inputId?: string;
+
+  analyticsLocation?: string;
```

Comment:
not a huge fan of having a prop just for analytics 😕

does rudderstack not just automatically include the page url where the interaction was emitted from? 😕 if not, we could maybe just use react-router's `useLocation` hook within `FolderPicker` and emit the page url that way? i think that would probably be good enough.

Reply:
you're right. page url is probably good enough.